src/OpenTelemetry.Extensions.AWS/OpenTelemetry.Extensions.AWS.csproj

```diff
@@ -17,7 +18,7 @@
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == '$(NetFrameworkMinimumSupportedVersion)'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
```

Comment:
Note: The `net8.0` target may get a vulnerable version of `System.Text.Json`. We should probably bump all targets <= `net8.0` to v8.0.4. I left it alone for this PR because we probably need to look into this repo-wide.

See: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5874

Reply:
/cc @Kielek @joegoldman2